compiler/stable_mir/src/mir/mono.rs

```diff
@@ -47,10 +48,23 @@ impl Instance {
         with(|context| context.instance_ty(self.def))
     }
 
-    pub fn mangled_name(&self) -> String {
+    pub fn mangled_name(&self) -> Symbol {
         with(|context| context.instance_mangled_name(self.def))
     }
 
+    pub fn name(&self) -> Symbol {
```

Comment:
Maybe add documentation to `name` and `mangled_name` as well, since we have too many functions with `name` in it, it can be easily confused which is which.